zme_radiotools.crontab.in

```diff
@@ -0,0 +1,3 @@
+#MAILTO=""
```

Comment:
Why not logrotate?

Reply:
logrotate can't work without rotation, but we need not rotation, we just need to remove old files.

Reply:
Also, support for SIGHUP (or something like this) is needed in scripts to logrotate work properly.

Reply:
ok